### ISSUE TITLE
 Admin cannot validate email from the user's profile view.

### ISSUE THREAD

**Issue by aashay8** [Haufe-Lexware/wicked.haufe.io#284]
Admin can validate email only from dropdown of pending verifications, not by visiting user's profile page.
Screenshots attached:
1. Admin's all user view page. 'Himanshu Local' is not yet verified after signing up
<img width="920" alt="Admin User list" src="https://user-images.githubusercontent.com/36197169/90365311-31427d00-e083-11ea-914d-0ea85d5c10d0.png">
2. Visiting Himanshu's User page. Top right shows the Admin ID
<img width="918" alt="User profile page of Not verified by mail" src="https://user-images.githubusercontent.com/36197169/90365345-3f909900-e083-11ea-908a-039343a65262.png">
3. On clicking the small hyperlink of validate (below the email address), we are redirected to verify url, which shows the user is already validated. **Tries to validate admin's own account, which is already validated**
<img width="933" alt="On clicking the validate link in user profile" src="https://user-images.githubusercontent.com/36197169/90365384-4fa87880-e083-11ea-8c43-03266680e9fb.png">

**Expected flow** :  validate the user whose profile is being visited.
Could anyone please tell if this is the intended flow ?

Thanks!

**Comment by DonMartin76** [comment]
True, this is currently not possible, and it has not been an intended use case/flow so far. But I understand the use case of it, and I think it would make sense to have this kind of functionality. I'll put it as "enhancement"; we would also of course accept pull requests. I could imagine something like a button which says "Mark user/email as valid" if you're logged in as an Admin, *and* if you're viewing some other profile than your own.

**Comment by aashay8** [comment]
Thank you for the reply @DonMartin76 !
